src/specs/index.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { testData } from './test-story-descriptions';
+import { extractLastReleaseMessage } from '../utils'
+
+
+describe('extractLastReleaseMessage', () => {
+    testData.forEach(

```

Comment:
I actually think it would be more readable to have the testData within the spec file, with a test for each different scenario. The test results will then be more explicit and tell me which scenario failed.

Reply:
SORRY - I can see that each scenario is running in its own test case. Ignore that comment :)

Reply:
👍 Done